### DISCUSSION TITLE
Can you create a single dynamo transaction with onetable operations and non-onetable operations?

### DISCUSSION THREAD

**Discussion by kjohnstonadept** [sensedeep/dynamodb-onetable#526] [Q&A]
Say I have some code that is using the same table as Onetable, but it has not yet been converted to use onetable. Is there a way to commit the onetable operations and non-onetable operations in the same transaction?

**Comment by mobsense** [comment]
Probably not. You could use the execute: false to prepare a OneTable command (See https://doc.onetable.io/api/model/params/#params). Then use that raw DynamoDB command with your non-onetable code in a transaction.
